content/en/docs/Tasks/update-operator-to-next-version.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Update operator to next version"
+date: 2021-03-09
+weight: 2
+description: >
+   Given an operator at version n, what should I do to create a version n+1.
+--- 
+
+## Introduction
+
+In the Operator Lifecycle Manager (OLM) ecosystem, the following resources are used to resolve Operator installations and upgrades:
+
+1. `ClusterServiceVersion (CSV)` - A YAML manifest created from Operator metadata that assists the Operator Lifecycle Manager (OLM) in running the Operator in a cluster.
+
+2. `CatalogSource` - Operator metadata, defined in CSVs, can be stored in a collection called a CatalogSource.
+
+3. `Subscription` - A user indicates a particular package and channel in a particular CatalogSource in a Subscription.
+
+
+OLM uses CatalogSources, which use the Operator Registry API, to query for available Operators as well as upgrades for installed Operators.
+
+![CatalogSource Image](/docs/Tasks/images/catalogsource.png)
+
+
+## Steps to update the operator
```

Comment:
IHMO we could explain that using SDK as well.
Also, if I am not wrong we have the feature sdk bundle upgrade to help on that.

Reply:
Following some suggestion:
- explain here that would recommend use semver. Note that the bundle operator name convention name is e.g `name: memcached-operator.v0.0.1`
- then the users need to push a new bundle with a new version 
- then, we can show how to do that manually and also with  SDK ( make bundle )
- then, also we, if I am not the wrong sdk, has a feature for upgrade bundles   
- also we can describe after manually by showing what is changed and where in this case